doc/rtd/topics/format.rst

```diff
@@ -89,6 +93,9 @@ Supported content-types are listed from the cloud-init subcommand make-mime:
 .. code-block:: shell-session
 
     $ cloud-init devel make-mime --list-types
+
```

Comment:
Could we add an `Example output:` line after the `cloud-init devel ...` command?

Reply:
+1